internal/controllers/machinedeployment/testdata/regular rollout, 3 replicas, maxsurge 0, maxunavailable 1.test.log.golden

```diff
@@ -0,0 +1,94 @@
+## Regular rollout, 3 Replicas, maxSurge 0, maxUnavailable 1
+
+[Test] Initial state
+  md, 3/3 replicas
+  - ms1, 3/3 replicas (m1,m2,m3)
+  - ms2, 0/0 replicas ()
+[Test] Rollout 3 replicas, MaxSurge=0, MaxUnavailable=1
+[MD controller] Iteration 1, Reconcile md
+[MD controller] - Input to rollout planner
+  md, 3/3 replicas
+  - ms1, 3/3 replicas (m1,m2,m3)
+  - ms2, 0/0 replicas ()
+[MD controller] - Result of rollout planner
+  md, 3/3 replicas
+  - ms1, 3/2 replicas (m1,m2,m3)
+  - ms2, 0/0 replicas ()
+[MS controller] Iteration 1, Reconcile ms1, 3/2 replicas (m1,m2,m3)
+[MS controller] - ms1 scale down to 2/2 replicas (m1 deleted)
+[MS controller] Iteration 1, Reconcile ms2, 0/0 replicas ()
+[MD controller] Iteration 2, Reconcile md
+[MD controller] - Input to rollout planner
+  md, 3/3 replicas
+  - ms1, 2/2 replicas (m2,m3)
+  - ms2, 0/0 replicas ()
+[MD controller] - Result of rollout planner
+  md, 2/3 replicas
+  - ms1, 2/2 replicas (m2,m3)
+  - ms2, 0/1 replicas ()
+[MS controller] Iteration 2, Reconcile ms1, 2/2 replicas (m2,m3)
+[MS controller] Iteration 2, Reconcile ms2, 0/1 replicas ()
+[MS controller] - ms2 scale up to 1/1 replicas (m4 created)
+[MD controller] Iteration 3, Reconcile md
+[MD controller] - Input to rollout planner
+  md, 2/3 replicas
```

Comment:
Why is this 2/3 and not 3/3? (similar below)

Is the rollout planer working with outdated information?

(but probably it doesn't matter because it is not looking at MD status, is that it? :))